app/src/main/java/zegreanu/cristi/criminalintent/CrimeListFragment.kt

```diff
@@ -0,0 +1,84 @@
+package zegreanu.cristi.criminalintent
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+
+class CrimeListFragment : Fragment() {
+    private lateinit var crimeRecyclerView: RecyclerView
+    private lateinit var adapter: CrimeAdapter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        arguments?.let { }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val rootView = inflater.inflate(R.layout.fragment_crime_list, container, false)
+
+        crimeRecyclerView = rootView.findViewById(R.id.crime_recycler_view)
+        crimeRecyclerView.layoutManager = LinearLayoutManager(activity)
+
+        activity?.let { updateUI(it) }
+
+        return rootView
+    }
+
+    private fun updateUI(context: Context) {
+        val crimeLab = CrimeLab.getInstance(context)
+        val crimes = crimeLab.getCrimes()
+
+        adapter = CrimeAdapter(crimes)
+        crimeRecyclerView.adapter = adapter
+    }
+
+    private class CrimeHolder(inflater: LayoutInflater, parent: ViewGroup) :
+        RecyclerView.ViewHolder(inflater.inflate(R.layout.list_item_crime, parent, false)) {
+
+        private var titleTextView: TextView = itemView.findViewById(R.id.crime_title)
+        private var dateTextView: TextView = itemView.findViewById(R.id.crime_date)
+        private lateinit var crime: Crime
+
+        init {
+            itemView.setOnClickListener {
+                Toast.makeText(
+                    itemView.context,
+                    crime.title + "CLICKED",
+                    Toast.LENGTH_SHORT
+                ).show()
+            }
+        }
+
+        fun bind(crimeItem: Crime) {
+            crime = crimeItem
+            titleTextView.text = crime.title
+            dateTextView.text = crime.date.toString()
+        }
+    }
+
+    private class CrimeAdapter(private val crimes: List<Crime>) :
+        RecyclerView.Adapter<CrimeHolder>() {
+
+        override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CrimeHolder {
+            return CrimeHolder(LayoutInflater.from(parent.context), parent)
```

Comment:
Let's call `inflate(R.layout.list_item_crime, parent, false)` here so we pass the itemView directly to `CrimeViewHolder`